todo_app/app.py

```diff
@@ -1,18 +1,25 @@
 from flask import Flask, redirect, render_template, request
-from todo_app.data.session_items import add_item, get_items
+from todo_app.data.trello_items import add_item, get_items, update_item
```

Comment:
Nice, the fact this is such a simple change here is a nice sign that you've kept your interfaces for each file clean!